### ISSUE TITLE
Picture Glance Card: camera_image with custom update interval

### ISSUE THREAD

**Issue by RyuzakiKK** [home-assistant/frontend#6509]
## The request

[Do not hardcode](https://github.com/home-assistant/frontend/blob/f8c5eeab5db76f2a14b6c8b51ddb045049d53599/src/panels/lovelace/components/hui-image.ts#L21) the `UPDATE_INTERVAL` value for the "Picture Glance Card", in this way the users will be able to set a custom update interval that fits their needs.

Currently "Picture Glance Card" has two modes to display the camera feed: using a static image or a live feed.

With a live feed you get a video stream of your camera, but the downside is that it's very resource intensive (it's a real video).

The other option is a static image. However the problem here is that the update interval is hardcoded to be 10 seconds, and for a camera feed a delay of 10 seconds might be too much.

## The alternatives

Apart from switching to the live stream mode (too bandwidth and CPU intensive for my use case) I'm not aware of any other alternatives.

Also apparently there aren't any custom component that can be used as a workaround

## Additional information

This is a rewrite of the previous issue https://github.com/home-assistant/frontend/issues/6040

**Comment by zsarnett** [comment]
https://github.com/home-assistant/frontend/pull/5466

This was PR'd here and turned down. I don't think this card will be any different

**Comment by RyuzakiKK** [comment]
Thanks for linking #5466, I wasn't aware that a PR have been already made.

It's unfortunate that it was rejected though.
Limiting the possible refresh rate to just integer values would mean that you can't set it faster than 1 refresh for every second. I think this was a good compromise.
And while this might slow down the UI in low powered devices, I doubt it would be enough to break the interface completely.
Of course if a user wants to open Home Assistant in a device that is not very powerful he needs to be careful with these options, but this is nothing new. Even currently, by just adding multiple widgets and/or multiple pictures with `camera_view: live`, will be enough to slow everything down considerably, e.g. if you open the UI in an old Android tablet.

**Comment by yorknh** [comment]
I agree. There's nothing wrong with having it default to 10s but the idea that someone would set that low enough that it slows down their system without them knowing that's why, seems pretty thin. Anyone looking to set a higher refresh rate would almost certainly check out the new refresh rate immediately after changing it, and would notice sluggish performance.

**Comment by markberndt** [comment]
10 seconds was seen as a performance ideal based on a test run in a particular year on specific silicon. 

There is another related picture card behavior that needs to be addressed before this can be a quality enhancement. 

After a view with picture cards is opened, each picture card is updated every update interval (10 seconds). The picture cards are not updated in the background when navigating off the view's tab... So far so good... however when switching back to the view with picture cards, the images are stale and will not be updated until a full update interval has elapsed. Not good, regardless of the update interval setting.

***Picture glance card images should refresh when coming into view if the update interval has elapsed while the card was running in the background.*** 

For example, a 5 minute update interval may promote UI stability and be adequate for a persistent object surveillance use case, but a wait of 5 minutes to view updated images after switching tabs to check for objects on demand would not be functional.
